src/utils/storeUtils/remoteListUtils.ts

```diff
@@ -0,0 +1,68 @@
+import { RemoteData, RemoteList, remoteList } from 'utils/storeUtils';
+
+/**
+ * When recieving response from backend that a remote list has been created, this method should be used to update the redux store (cache).
+ * This is a utility method for manipulating the cache, and should only be used in this context.
+ *
+ * @returns the created RemoteList.
+ */
+export function remoteListCreated<
+  DataType extends RemoteData
+>(): RemoteList<DataType> {
+  const list = remoteList<DataType>();
+  list.loaded = new Date().toISOString();
+  return list;
+}
+
+/**
+ * When recieving response from backend that a remote list is being loaded, this method should be used to update the redux store (cache).
+ * This is a utility method for manipulating the cache, and should only be used in this context.
+ *
+ * @param list RemoteList to set as loading.
+ * @returns the RemoteList.
+ */
+export function remoteListLoad<DataType extends RemoteData>(
+  list: RemoteList<DataType> | null
+): RemoteList<DataType> {
+  if (!list) {
+    list = remoteList();
+  }
+  list.isLoading = true;
+  return list;
+}
+
+/**
+ * When recieving response from backend that a remote list has been loaded, this method should be used to update the redux store (cache).
+ * It receives a list of items, and makes sure a list is created with the loaded items. Returns the list.
+ * This is a utility method for manipulating the cache, and should only be used in this context.
+ *
+ * @param items Loaded RemoteItem-compatible data used for populating the list.
+ * @returns the loaded RemoteList.
+ */
+export function remoteListLoaded<DataType extends RemoteData>(
+  items: DataType[]
+): RemoteList<DataType> {
+  const list = remoteList(items);
+  const timeStamp = new Date().toISOString();
+  list.loaded = timeStamp;
+  list.items.forEach((item) => (item.loaded = timeStamp));
+  return list;
+}
+
+/**
+ * When recieving response from backend that a remote list has been invalidated, this method should be used to update the redux store (cache).
```

Comment:
The invalidated-callbacks are triggered by the user's action, not by a message from backend.

```suggestion
 * When a remote list has been invalidated, this method should be used to update the redux store (cache).
```

Reply:
On 3f9b5223541f82b9a55a5b5e07e716f3e35d0592, the documenation for `remoteListInvalidated` still says "When recieving response from backend that a remote list has been invalidated, ..."